public/video-ui/src/components/ManagedForm/ManagedForm.js

```diff
@@ -23,6 +24,13 @@ export class ManagedForm extends React.Component {
     }
   }
 
+  getFormClass = () => {
+    if (React.Children.toArray(this.props.children).some(child => child.type.componentType === 'managedField')) {
```

Comment:
For some reason I made the type of  `managedSection` `managedField`so I'm going to change that. But I'm not sure if there is a simpler way of doing this, `managedForm` needs to know about the type of its children to know its css class.